src/exception_page.cr

```diff
@@ -39,7 +39,7 @@ abstract class ExceptionPage
     @headers = context.response.headers.to_h
     @method = context.request.method
     @path = context.request.path
-    @url = "#{context.request.host_with_port}#{context.request.path}"
+    @url = "#{context.request.headers["Host"]?}#{context.request.path}"
```

Comment:
This was a deprecation warning originally added in https://github.com/crystal-loot/exception_page/pull/18 but since specs were failing, I just added it here.